test/libsolidity/syntaxTests/pragma/valid_range_operator_two_ranges.sol

```diff
@@ -0,0 +1,2 @@
+pragma solidity <0.8.15 || 0.8.17 - 0.8.20;
```

Comment:
This test has only one range, but the name says it has two. It should be called something like `valid_range_operator_range_second_prefixed_version.sol`.